### DISCUSSION TITLE
Houdini and Docker

### DISCUSSION THREAD

**Discussion by BrianHanke** [dreamworksanimation/openmoonray#38] [Q&A] [answered]
Taking a break from my Windows tests, I built MoonRay using Docker on a RHEL 9 VM. Super easy process, took maybe 45 minutes total. Test images are rendering just fine.

Is it possible to use hdmoonray in Houdini Solaris when MoonRay lives in Docker? Not sure if/how Docker can interface with the surrounding OS. Thanks!

**Comment by rgwilson68** [comment, marked as answer]
No, I don't think there's any way to do this. The hdMoonray.so plugin is loaded into the Houdini process, so it would need to be built for the same OS environment as Houdini.

**Comment by BrianHanke** [comment]
Makes sense, thanks!

**Reply by panda1100** [reply]
@BrianHanke Apptainer (container solution for HPC) may work with some tweaks. This is interesting topic for me.
